hw2/controllers/UserControllers.js

```diff
@@ -46,7 +47,8 @@ export const getUsersByID = async (req, res) => {
 export const postUser = async (req, res) => {
   try {
     const { login, password, age } = req.body;
-    const newUser = await User.create({ login, password, age });
+    const id = uuidv4();

```

Comment:
move this to beforeCreate hook
https://sequelize.org/docs/v6/other-topics/hooks/